modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/db/Statement.scala

```diff
@@ -55,7 +55,7 @@ object Statement {
   case class FoldersCopy(source: S3.Folder) extends Statement
 
   // Loading
-  case class EventsCopy(path: S3.Folder, compression: Compression) extends Statement with Loading {
+  case class EventsCopy(path: S3.Folder, compression: Compression, columns: List[String] = List.empty) extends Statement with Loading {
```

Comment:
I'm a bit uncomfortable about this default `List.empty`, and they way you leave it empty for Redshift.  I understand you don't need the columns for Redshift, but if you leave it empty then the `columns` field has ambiguous meaning.

So I almost suggested to set the field anyway, even though you don't need it.

But then I remembered in the Databricks Loader you cannot possibly set the field, because we don't know the columns.

So I don't have a good suggestion!  Let's leave it as it is.

Reply:
Maybe for clarity you could remove the default `List.empty`.  And then in the redshift module, when you explicitly set it to `Nil` then you can have a code comment explaining why it's not needed.

Just an idea though, it's fine if you ignore this.